tools/release/java/build.xml

```diff
@@ -14,35 +14,31 @@
 <project name="release-tools" default="tools" basedir="." >
     <target name="init">
         <tstamp/>
-        <property name="src.dir" value="src"/>
-        <property name="lib.dir" value="lib"/>
-        <property name="build.dir" value="classes"/>
         <property file="api-report.properties" />
-
-        <mkdir dir="${build.dir}"/>
+        <property name="apireport.jar" location="target/icu4c-apireport.jar"/>
+        <property name="rsrc.dir" location="${basedir}/src/main/resources"/>
         <echo message="java home: ${java.home}"/>
         <echo message="java version: ${java.version}"/>
         <echo message="java vendor: ${java.vm.vendor}"/>
         <echo message="ant java version: ${ant.java.version}"/>
         <echo message="${ant.version}"/>
+        <echo message="tools jar=${apireport.jar}" />
         <echo message="${basedir}"/>
+        <condition property="apireport.jar.exists">
+          <available file="${apireport.jar}" type="file"/>
+        </condition>
     </target>
 
-    <target name="tools"  depends="init" description="compile release tools">
-        <javac includeantruntime="false" includes="**/*.java"
-            srcdir="${src.dir}"
-            destdir="${build.dir}"
-            source="1.7"
-            debug="on" deprecation="off"
-            encoding="utf8"/>
+    <target name="tools"  depends="init" description="compile release tools" unless="apireport.jar.exists">
+      <exec command="mvn package"/> <!-- complains, but it's cross platform -->
```

Comment:
I use:

        <exec executable="mvn" searchpath="true">
            <arg value="compile"/>
        </exec>

Which might be neater (the binary is clearly delimited from the arguments).

Reply:
this is a great suggestion.